<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TranslateData</class>
 <widget class="QDialog" name="TranslateData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Translate Data</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-100</x>
     <y>260</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="meshListBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="meshListLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>110</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>110</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>   Select mesh:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>281</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Translation Vector</string>
   </property>
   <widget class="QLineEdit" name="xlineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ylineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="zlineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="xLengthLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>16</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="yLengthLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>16</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QLabel" name="zLengthLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>16</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>z</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="geoListLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>110</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>110</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>110</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>   Select geo:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="geoListBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TranslateData</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TranslateData</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
